src/components/Form/Form.js

```diff
@@ -1,43 +1,43 @@
 import React, { Component } from 'react';
 
 class Form extends Component {
-  
-submitForm = event => {
-  event.preventDefault();
 
-  const {
-    header,
-    users
-  } = this.props.state;
-  const form = event.target;
-  const formInputName = form.name.value;
-  const formInputSurname = form.surname.value;
-  const formInputLocation = form.location.value;
-  const formInputAge = form.age.value;
-  const randomId = Math.floor(Math.random() * 10000);
+  submitForm = event => {
+    event.preventDefault();
 
-  const newState = {
-    header: [...header],
-    users: [
-      ...users,
-      {
-        id: `${randomId}`,
-        name: `${formInputName}`,
-        surname: `${formInputSurname}`,
-        location: `${formInputLocation}`,
-        age: `${formInputAge}`
-      }
-    ]
-  };
+    const {
+      header,
+      users
+    } = this.props.state;
+    const form = event.target;
+    const formInputName = form.name.value;
+    const formInputSurname = form.surname.value;
+    const formInputLocation = form.location.value;
+    const formInputAge = form.age.value;
```

Comment:
tak nie może być ze z forma wyciagasz wszystkie dane po kolei. Dlatego prosze zebys zapisał te dane do state.

bo potem w tej metodzie bedziesz mial tak 
`const {nameValue, surnameValue, locationValue, ageValue} = this.state;    `
masz jedną linijkę zamiast kilku.